prow.sh

```diff
@@ -879,8 +879,10 @@ install_sanity () (
         return
     fi
 
-    git_checkout "${CSI_PROW_SANITY_REPO}" "${GOPATH}/src/${CSI_PROW_SANITY_IMPORT_PATH}" "${CSI_PROW_SANITY_VERSION}" --depth=1 || die "checking out csi-sanity failed"
-    run_with_go "${CSI_PROW_GO_VERSION_SANITY}" go test -c -o "${CSI_PROW_WORK}/csi-sanity" "${CSI_PROW_SANITY_IMPORT_PATH}/cmd/csi-sanity" || die "building csi-sanity failed"
+    git_checkout "${CSI_PROW_SANITY_REPO}" "${GOPATH}/src/${CSI_PROW_SANITY_PACKAGE_PATH}" "${CSI_PROW_SANITY_VERSION}" --depth=1 || die "checking out csi-sanity failed"
+    cd "${GOPATH}/src/${CSI_PROW_SANITY_PACKAGE_PATH}/cmd/csi-sanity" || die "enter csi-sanity directory failed"
+    go build || die "building csi-sanity failed"
+    mv ./csi-sanity "${CSI_PROW_WORK}"
```

Comment:
Don't call `go` directly, always wrap it with `run_with_go` to ensure that the right version of Go is in the `PATH`. Permanently changing the directory can also be confusing. Better use:
```
git_checkout "${CSI_PROW_SANITY_REPO}" "${GOPATH}/src/${CSI_PROW_SANITY_PACKAGE_PATH}" "${CSI_PROW_SANITY_VERSION}" --depth=1 || die "checking out csi-sanity failed"
( cd "${GOPATH}/src/${CSI_PROW_SANITY_PACKAGE_PATH}/cmd/csi-sanity" && run_with_go "${CSI_PROW_GO_VERSION_SANITY}" go build -o "${CSI_PROW_WORK}/csi-sanity" ./cmd/csi-sanity ) || die "building csi-sanity failed"
```

Reply:
Fixed

Reply:
```
Wed Jan 20 03:46:22 UTC 2021 go1.15.5 /home/prow/go/src/github.com/kubernetes-csi/csi-test/cmd/csi-sanity$ go build -o /home/prow/go/pkg/csiprow.Fc5cBsm6I0/csi-sanity ./cmd/csi-sanity
stat /home/prow/go/src/github.com/kubernetes-csi/csi-test/cmd/csi-sanity/cmd/csi-sanity: directory not found
ERROR: building csi-sanity failed
```

Maybe `( cd "${GOPATH}/src/${CSI_PROW_SANITY_PACKAGE_PATH}/cmd/csi-sanity" && run_with_go "${CSI_PROW_GO_VERSION_SANITY}" go build -o "${CSI_PROW_WORK}/csi-sanity") || die "building csi-sanity failed"` is correct command

Reply:
Either changing into `cmd/csi-sanity` and `go build` without parameters or changing into the root and then using `go build ./cmd/csi-sanity` work, but not doing both :grin: 

I don't have a preference either way and verified that that the current commit works, so let's use that.

/lgtm
/approve